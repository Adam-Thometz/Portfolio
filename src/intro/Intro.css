.Intro {
  /* border: 1px solid black; */
  border-radius: 50px;
  background-image: linear-gradient(rgb(95, 40, 40), rgb(24, 63, 92));
  height: 80%;
  box-shadow: 5px 10px 20px 20px rgb(95, 40, 40),
    inset 5px 10px 20px 20px rgb(24, 63, 92);
}

.Intro-header-animation-container {
  display: flex;
  justify-content: space-between;
  height: 70%;
}

.Intro-header {
  width: 50%;
  padding: auto 0;
  margin: auto 20px;
}

.Intro-animation {
  width: 50%;
  border: none;
  margin: auto 20px;
}