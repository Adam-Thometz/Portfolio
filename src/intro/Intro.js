import React from "react";

import "./Intro.css";

import { ME } from "../_media/assets";

const Intro = ({ style }) => {
  return (
    <header className="Intro" style={style}>
      <section className="Intro-top">
        <aside>
          <h1>Hi, I'm Adam! 👋🏼</h1>
          <h2>I am a full-stack developer and music producer based in NYC.</h2>
        </aside>
        <img src={ME} alt="the developer" />
      </section>
    </header>
  );
};

export default Intro;
