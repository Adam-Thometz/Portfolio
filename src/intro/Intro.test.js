import React from "react";
import { render } from "@testing-library/react";
import Intro from './Intro';

describe('Intro component', () => {
  it('renders without crashing', () => {
    render(<Intro />);
  });

  it('matches the snapshot of the landing page displaying the games page', () => {
    const { asFragment } = render(<Intro />);
    expect(asFragment()).toMatchSnapshot();
  });
});