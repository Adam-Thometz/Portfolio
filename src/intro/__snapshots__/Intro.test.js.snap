// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro component matches the snapshot 1`] = `
<DocumentFragment>
  <header
    class="Intro"
  >
    <h1>
      Hi, I'm Adam! 👋🏼
    </h1>
    <h2>
      I am a full-stack developer specializing in frontend technologies based in NYC.
    </h2>
    <p>
      As a musician and former educator, I love discovering patterns to make code more efficient and creating delightful and intuitive UIs.
    </p>
    <b>
      Click on a project below to learn more.
    </b>
  </header>
</DocumentFragment>
`;
